### PR TITLE
Clean Up Bugs In Schedule

### LINKED ISSUE #138: Figure out bug to show pod schedule
Currently the pod schedule isnt being shown when the toggle is clicked

### PR DESCRIPTION
**Pull request by allysmith12**
- Fixes Getting avatars and users for pod in the schedule 
- Fixes toggling to the pod schedule 
- Adds avatars to the pod schedule page

### CODE REVIEW

File: api/src/routes/users.ts [hunk 1]
```diff
@@ -159,11 +159,11 @@ usersRouter.get("/avatars", [auth], async (req: Request, res: Response) => {
...
 
-// This is a hack and should be temporary!
-usersRouter.get("/email/:id", [auth], async (req: Request, res: Response) => {
+// Get email of current user
+usersRouter.get("/email", [auth], async (req: Request, res: Response) => {
   try {
-    const { id } = req.params;
-    const user = await User.query().findOne("id", id);
+    const userId = (req as AuthRequest).user.id;
+    const user = await User.query().findOne("id", userId);
```
Comment (zusoomro) [inline api/src/routes/users.ts:166]: Nice fix! Definitely better

File: mobile/src/events/eventsSlice.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ const eventSlice = createSlice({
   initialState,
   reducers: {
     setEvents(state, action) {
-      console.log(action.payload);
+      // console.log(action.payload);
```
Comment (zusoomro) [inline mobile/src/events/eventsSlice.ts]: Maybe just take this out completely?

**Review by zusoomro — APPROVED** [review]
Looks good, great fix!

File: mobile/src/events/eventsSlice.ts (hunk 2 above)
Comment (allysmith12) [reply mobile/src/events/eventsSlice.ts]: Can do!

**Pull request merged**